README.md

```diff
@@ -40,3 +40,22 @@ Trello -13% error
 
 ## Note
 - background has a different environment from popup, scope is not shared
+- When improperly formatted data has been written to storage, the extension will not work properly. To fix this, 
+  - remove the extension and reinstall it, or
+  - call `remoteRepository.clear()` in popup.tsx, e.g.
+  ```typescript
+  useMountEffect(() => {
+        remoteRepository.clear().then(() => {
+            remoteRepository
+                .get({ currentPage: "landing" })
+                .then((data) => data.currentPage as Page)
+                .then(async (storedPage) => {
+                    if (storedPage === "results") {
+                        await refreshCalculationHistory();
+                    }
+                    setPage(storedPage);
+                });
+        });
+    });
```

Comment:
should this not be something called in the cleanup of useMountEffect ?

Reply:
actually no based on your description the start does make sense - would be keen to hear more about the issue though

Reply:
I think this other approach is more confusing then beneficial - will remove it from the README

the original code looks like this:
```typescript
useMountEffect(() => {
            remoteRepository
                .get({ currentPage: "landing" })
                .then((data) => data.currentPage as Page)
                .then(async (storedPage) => {
                    if (storedPage === "results") {
                        await refreshCalculationHistory();
                    }
                    setPage(storedPage);
                });
    });
```
without `remoteRepository.clear().then(...`

It's just a code based alternative to reinstalling the extension, with the goal of wiping the database

Reply:
i see that - but is it something that needs be run before ot after the session ? if it needs to be wiped after the session its probably best to use a cleanup function - but if you wanna do it before just to make sure that also makes sense

Reply:
- Nope, `remoteRepository.clear()` is not something that should be called on a normal basis
  - `remoteRepository.clear()` wipes the database for that user
- it's a one-off invocation for devs, 
  - e.g. if you accidentally write unwanted formats to the database in dev
    - then this will cause errors in the app
    - so you need to wipe the database 

Do you think this setup can be improved?

Reply:
im just trying to get an understanding of how its used - so if a new user clicks on the extension it wipes the database, and if an existing user reboots the extension it also wipes the data ?

Reply:
- if any user clicks on the extension, 
  - the database should not be wiped
- if an existing user reboots the extension
  - database is wiped by chrome automatically